src/authorization.ts

```diff
@@ -1,31 +1,14 @@
 import * as _ from 'lodash';
 
-import {
-  AuthType,
-  AddressHashMode,
-  PubKeyEncoding,
-  RECOVERABLE_ECDSA_SIG_LENGTH_BYTES
-} from './constants'
-
-import {
-  BufferArray,
-  BufferReader,
-  txidFromData,
-  sha512_256
-} from './utils';
-
-import {
-  Address
-} from './types';
-
-import {
-  StacksPublicKey,
-  StacksPrivateKey
-} from './keys';
-
-import {
-  StacksMessage
-} from './message'
+import { AuthType, AddressHashMode, PubKeyEncoding, RECOVERABLE_ECDSA_SIG_LENGTH_BYTES } from './constants';
```

Comment:
I would prefer if we can keep a line length maximum. For me it really helps with readability. Is there any reason not to?

Reply:
This is a Prettier rule we have set at 120.

We upped it from the default, 80, as with TypeScript it really isn't enough. With hindsight, I'm now of the opinion 120 is too long, and ~100 would be optimal.

Reply:
100 is good enough for me

Reply:
Thanks @yknl , good suggestion! I've updated the local config for this project and fixed any resulting long lines. I would agree that the 100 char version is better.

Reply:
https://github.com/blockstack/prettier-config/pull/1